extra/changelog-generator/changelog-generator

```diff
@@ -161,6 +162,36 @@ def get_range_for_repo(repo, range):
     )
 
 
+def strip_type(message, fixes, feats, others):
```

Comment:
Q:
Just looking at it, it's a little odd to me that a `strip_type` function returns anything else than what it has stripped.

It's no biggie, but I would not consider it exactly a clean API.

Reply:
It returns the stripped message:
`return stripped_message, bucket`

Reply:
I think I would agree. It's mostly the name, the function itself is fine. How about `extract_message_and_bucket`?

Reply:
Oh yeah, I can see that it could be interpreted that way :)